src-input/duk_hobject.h

```diff
@@ -36,6 +36,11 @@
  * inspection code.
  */
 
+/* FIXME: ES spec: all construct-only functions have [[Call]] and [[Construct]]
```

Comment:
I understood, I was just trying to figure what it meant in practice--since if there's no observable difference in behavior then it doesn't matter for compliance.  But I guess in this case the difference would be that the thrown error would not have the function in its stack trace, while it would in a compliant engine.